### PR TITLE
Implement dao withdraw

### PR DESCRIPTION
**Pull request by duanyytop**

### CODE REVIEW

File: ckb/src/main/java/org/nervos/ckb/type/cell/CellWithStatus.java [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 /** Copyright © 2019 Nervos Foundation. All rights reserved. */
 public class CellWithStatus {
+  public static final String LIVE = "live";
```
Comment (quake) [inline ckb/src/main/java/org/nervos/ckb/type/cell/CellWithStatus.java]: suggest creating an enum for status: live/dead/unknown

File: ckb/src/main/java/org/nervos/ckb/type/transaction/TransactionWithStatus.java [hunk 2]
```diff
@@ -4,6 +4,9 @@
 
 /** Copyright © 2019 Nervos Foundation. All rights reserved. */
 public class TransactionWithStatus {
+  public static final String PENDING = "pending";
```
Comment (quake) [inline ckb/src/main/java/org/nervos/ckb/type/transaction/TransactionWithStatus.java]: suggest creating an enum for status

File: example/src/main/java/org/nervos/ckb/NervosDaoExample.java [hunk 3]
```diff
@@ -9,55 +9,98 @@
...
+        String withdrawTxHash = args[1];
+        TransactionWithStatus withdrawTx = api.getTransaction(withdrawTxHash);
+        OutPoint depositOutPoint = withdrawTx.transaction.inputs.get(0).previousOutput;
+        OutPoint withdrawOutPoint = new OutPoint(withdrawTxHash, "0x0");
+        Transaction transaction =
+            generateClaimingFromDaoTx(depositOutPoint, withdrawOutPoint, Utils.ckbToShannon(0.01));
+        String txHash = api.sendTransaction(transaction);
+        System.out.println("Nervos DAO withdraw phase2 tx hash: " + txHash);
+        // Waiting some time to make tx into blockchain
+        System.out.println("After depositing, balance: " + getBalance(DaoAddress) + " CKB");
```
Comment (quake) [inline example/src/main/java/org/nervos/ckb/NervosDaoExample.java]:
```suggestion
        System.out.println("After withdrawal, balance: " + getBalance(DaoAddress) + " CKB");
```

File: ckb/src/main/java/org/nervos/ckb/type/cell/CellWithStatus.java (hunk 1 above)
Comment (duanyytop) [reply ckb/src/main/java/org/nervos/ckb/type/cell/CellWithStatus.java]: Updated. https://github.com/nervosnetwork/ckb-sdk-java/pull/252/commits/c2871cfe874d0ca6966309dcc2d43afd2d145421

File: ckb/src/main/java/org/nervos/ckb/type/transaction/TransactionWithStatus.java (hunk 2 above)
Comment (duanyytop) [reply ckb/src/main/java/org/nervos/ckb/type/transaction/TransactionWithStatus.java]: Updated. https://github.com/nervosnetwork/ckb-sdk-java/pull/252/commits/c2871cfe874d0ca6966309dcc2d43afd2d145421

File: example/src/main/java/org/nervos/ckb/NervosDaoExample.java (hunk 3 above)
Comment (duanyytop) [reply example/src/main/java/org/nervos/ckb/NervosDaoExample.java]: Updated. https://github.com/nervosnetwork/ckb-sdk-java/pull/252/commits/c2871cfe874d0ca6966309dcc2d43afd2d145421

**Pull request merged**